pentasharp/Models/Entities/Taxi.cs

```diff
@@ -23,7 +23,7 @@ public class Taxi
         [StringLength(50)]
         public string DriverName { get; set; } = string.Empty;
 
-        [Required]
+        [Required]

```

Comment:
remove data annotations as we agreed on removing them, add missing summaries for the whole model. rename this Taxi model to the corresponding TaxiRequest or TaxiResponse and the folder shouldnt be named Entities